docs/product-installation-guide/src/main/asciidoc/controller-eap-run-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='controller-eap-run-proc']
+= Running the standalone {KIE_SERVER} Controller
+After you have installed the standalone {KIE_SERVER} Controller on {EAP}, use this procedure to run the standalone {KIE_SERVER} Controller.
+
+.Prerequisite
+The standalone {KIE_SERVER} Controller installed and configured in an `__EAP_HOME__`
+
+.Procedure
+. In a terminal application, navigate to `__EAP_HOME__/bin`.
+. Enter the following command:
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./standalone.sh
+----
+** On Windows:
++
+[source,bash]
+----
+standalone.bat
+----
+. To verify that the Controller is working on {EAP}, enter the following command where `<CONTROLLER_USER>` and `<CONTROLLER_PWD>` is the user name and password combination that you created in <<controlloer-eap-users-create-proc>>. The output of this command provides information about the {KIE_SERVER} instance.
++
+[source]
+----
+curl -X GET "http://<HOST>:<PORT>/controller/rest/controller/management/servers" -H  "accept: application/xml" -u '<CONTROLLER>:<CONTROLLER_PWD>'
+----
+
+. In a web browser, enter the value of the `org.kie.server.controller` property, for example `http://localhost:8080/kie-wb/rest/controller`.
```

Comment:
Please delete this point, the controller functionality was checked in previous step.